workshops/management/commands/fake_database.py

```diff
@@ -39,10 +39,18 @@ def fake_airports(self, faker, count=5):
 
     def fake_roles(self, faker):
         """Provide fixed roles (before they end up in fixtures, see #626)."""
-        roles = ['helper', 'instructor', 'host', 'learner', 'organizer',
-                 'tutor', 'debriefed']
-        for role in roles:
-            Role.objects.create(name=role)
+        roles = [
+            ('debriefed', 'Debriefed', False),
```

Comment:
I don't know. This role was introduced along with all other roles here https://github.com/swcarpentry/amy/commit/2942f0ed9c8f192811eaf2025d0c5d2cdf3d745b